### ISSUE TITLE
Does trianing the model needs h5 file?

### ISSUE THREAD

**Issue by william-chen-zy** [chaneyddtt/Convolutional-Sequence-to-Sequence-Model-for-Human-Dynamics#10]
Hi, I am trying to train the model by doing python src/AC_main.py, however I run into the error OSError: Unable to create file (unable to open file: name = './samples/CNNAdTrain_GANWEIGHT0.010000_Sampling0.950000WindownLength20-2000.h5', errno = 2, error message = 'No such file or directory', flags = 15, o_flags = c2). 

After reading rest of the issues I get confused about the .h5 file. Do I need to create it by running test first? or should this file be automatically generated when I run python src/AC_main.py?

Thanks

**Comment by william-chen-zy** [comment]
Add: 
in the Autoencoder_gan.py, why do we need to call inferenceSample in the Train method? Seems this is causing the problem during training.

![image](https://user-images.githubusercontent.com/48894264/139594955-0757539a-8178-4ff2-999b-4c89c9c5c609.png)

**Comment by chaneyddtt** [comment]
Hi @william-chen-zy, we call the inferenceSample because we want to test our model every 2000 iterations. You might need to create a file named 'samples', where the saved model will be located in. Hope this helps.

**Comment by william-chen-zy** [comment]
Hi @chaneyddtt thank you for your prompt response. I commented out the self.InferenceSample line and was able to train the model successfully. My understanding is that in order to do Inference during the training (test model every 2000 iterations) I will need to have a trained model first. Am I right?

**Comment by chaneyddtt** [comment]
Yes, you can run the inference step after you have a trained model.  Basically, the inference step is to compute error for each action and also save the predicted sequence for further visualization.
